### PR TITLE
Add vista2d tutorial

### PR DESCRIPTION
**Pull request by yiheng-wang-nv**

### CODE REVIEW

File: notebooks/Training a VISTA2d Bundle.ipynb [hunk 1]
```diff
@@ -0,0 +1,615 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Training a MONAI Segmentation Bundle\n",
```
Comment (mingxin-zheng) [inline notebooks/Training a VISTA2d Bundle.ipynb:8]: Please change the title

File: notebooks/Training a VISTA2d Bundle.ipynb [hunk 2]
```diff
@@ -0,0 +1,615 @@
...
+    "\n",
+    "This tutorial demonstrates how to train VISTA2d bundle onthe NVIDIA DGX Cloud. It focuses on utilizing the powerful capabilities of DGX systems for medical imaging applications, specifically using a MONAI Vista bundle.\n",
```
Comment (mingxin-zheng) [inline notebooks/Training a VISTA2d Bundle.ipynb:10]: onthe typo

File: notebooks/Training a VISTA2d Bundle.ipynb [hunk 3]
```diff
@@ -0,0 +1,615 @@
...
+    "print(json.dumps(res, indent=2))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Experiment Creation\n",
+    "\n",
+    "Create a MONAI segmentation experiment, specifying the necessary parameters and datasets. In this tutorial, we will use the vista3d bundle."
```
Comment (Nic-Ma) [inline notebooks/Training a VISTA2d Bundle.ipynb:278]: vista3d -> vista2d

**Review by Nic-Ma — APPROVED** [review]
others look good to me.

Thanks.

**Pull request merged**